src/waka-realtime/index.ts

```diff
@@ -0,0 +1,70 @@
+import { pRateLimit, QuotaManager, RedisQuotaManager, Quota } from 'p-ratelimit'
+
+import Redis from './Redis'
+import createLogger from './logger'
+
+import { isKeyof } from '../utils'
+import { Logger } from '../typings'
+
+import BaseRealtime from './BaseRealtime'
+
+import AucklandRealtime from './regions/nz-akl'
+import SydneyRealtime from './regions/au-syd'
+
+const Regions = {
+  'au-syd': SydneyRealtime,
+  'nz-akl': AucklandRealtime,
+}
+
+interface RealtimeConfig {
+  prefix: string
+  quota?: Quota
+  version: string
+  api: { [prefix: string]: string }
+}
+
+class Realtime {
+  redis: Redis
+  prefix: string
+  quotaManager: QuotaManager
+  rateLimiter: <T>(fn: () => Promise<T>) => Promise<T>
+  region: BaseRealtime
+  logger: Logger
+  constructor(config: RealtimeConfig) {
+    const logger = createLogger(config.prefix, config.version)
+    this.logger = logger
+    this.prefix = config.prefix
```

Comment:
I would probably also prepend a waka-rt- to the key too? Just in case we need to use it in other components.